### PR TITLE
First Commit of Unifi App to marketplace

### PR DESCRIPTION
**Pull request by keithhubner**
Thank you for wanting to submit a Pull Request to the Civo Kubernetes Marketplace repository!

If your pull request is to submit a new application to the marketplace, please answer the following questions:

* [ ] Have you followed the guidelines in our Contributing document in setting up the application?
* [ ] Have you checked that the application your submission is proposing to add does not yet exist on the Marketplace, or as a pull request to be processed?
* [ ] Have you tested your application on the Civo managed Kubernetes service? Please include a screenshot.
* [ ] Can you confirm that the software you are submitting licensed for use in a publicly-available Kubernetes marketplace?

If your pull request concerns an existing Marketplace application, please make sure you have:

* [ ] Notified the Maintainer of the application in this pull request so that they are aware of your proposal.
* [ ] Outlined the changes you are proposing, and the reasons these are required (e.g. stability, compatibility with new versions of Kubernetes implementations, etc).
* [ ] Tested that the application works with the proposed updates applied (including a screenshot)

### CODE REVIEW

**Comment by keithhubner** [comment]
Updated logo file

File: unifi-controller/post_install.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+UDP 	3478 	Port used for STUN.
+TCP 	8080 	Port used for device and controller communication.
+TCP 	8443 	Port used for controller GUI/API as seen in a web browser
+TCP 	8880 	Port used for HTTP portal redirection.
+TCP 	8843 	Port used for HTTPS portal redirection.
+TCP 	6789 	Port used for UniFi mobile speed test.
+
+>  NOTE: These ports need to be open at the gateway/firewall as well as on the controller host. This would be achieved by creating port forwards on the gateway/firewall where the controller host is located.
+
+There is a guide on how to setup the controller with Traefik2 here:
```
Comment (saiyam1814) [inline unifi-controller/post_install.md:18]: the link is missing here ?

Comment (saiyam1814) [reply unifi-controller/post_install.md:18]: Also steps to access the UI are missing

**Comment by saiyam1814** [comment]
Left a few comments @keithhubner

File: unifi-controller/post_install.md (hunk 1 above)
Comment (keithhubner) [reply unifi-controller/post_install.md:18]: OK no problem, i'll need to get the guide done and approved first.

Comment (saiyam1814) [reply unifi-controller/post_install.md:18]: ok I can approve it, after that can you add steps to just access the application?

File: unifi-controller/manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+name: Unifi-network-controller
```
Comment (saiyam1814) [inline unifi-controller/manifest.yaml:1]: name should be all small and label you can add any 
Example 
```
name: falco-security
title: "Falco Security"
```

**Comment by DoNnMyTh** [comment]
@keithhubner Please make the requested changes ^

File: unifi-controller/manifest.yaml (hunk 2 above)
Comment (saiyam1814) [reply unifi-controller/manifest.yaml:1]: @keithhubner can you make it like below 
```
name: unifi-network-controller
title: "Unifi network controller"
version: "stable-6"
```

Comment (saiyam1814) [reply unifi-controller/manifest.yaml:1]: @keithhubner I cn merge once this is resolved

Comment (keithhubner) [reply unifi-controller/manifest.yaml]: Thanks, i've updated those bits, once it's in the marketplace I can do a full guide installing from the marketplace and also adding in an example of deploying with Traefik2 (hopefully also from the marketplace).

**Review by DoNnMyTh — APPROVED** [review]
Can be merged now @saiyam1814

**Comment by saiyam1814** [comment]
LGTM

**Pull request merged**